src/libs/EmojiTrie.ts

```diff
@@ -1,29 +1,46 @@
-import _ from 'underscore';
+import React from 'react';
+import {SvgProps} from 'react-native-svg';
 import emojis, {localeEmojis} from '../../assets/emojis';
 import Trie from './Trie';
 import Timing from './actions/Timing';
 import CONST from '../CONST';
+import {MetaData} from './Trie/TrieNode';
+
+type Emoji = {
+    code: string;
+    header?: boolean;
+    icon?: React.FC<SvgProps>;
```

Comment:
@fabioh8010, do we use React.FC or React.Element, React.Node?

Reply:
@rezkiy37 I checked and it's correct this way because our `.svg` are being typed as `React.FC<SvgProps>` in `global.d.ts`.